crates/oxc_query/src/lib.rs

```diff
@@ -0,0 +1,170 @@
+#![feature(let_chains)]
+#![allow(clippy::redundant_pub_crate)]
+mod adapter;
+mod edges;
+mod entrypoints;
+mod properties;
+mod util;
+mod vertex;
+
+pub use adapter::{schema, Adapter};
+pub use vertex::Vertex;
+
+// TODO: Uncomment on next release of trustfall
+#[cfg(test)]
+mod test {
+    use std::{collections::BTreeMap, rc::Rc, sync::Arc};
+
+    use oxc_allocator::Allocator;
+    use oxc_parser::Parser;
+    use oxc_semantic::SemanticBuilder;
+    use oxc_span::SourceType;
+    use trustfall::{execute_query, FieldValue, TryIntoStruct};
+
+    use crate::{adapter::schema, Adapter};
+
+    #[test]
+    fn test_path_query() {
+        let code = "const apple = 1;";
+
+        let allocator = Allocator::default();
+        let source_type =
+            SourceType::default().with_module(true).with_jsx(true).with_typescript(true);
+        let ret = Parser::new(&allocator, code, source_type).parse();
+        let program = allocator.alloc(ret.program);
+        let semantic_ret =
+            SemanticBuilder::new(code, source_type).with_trivias(&ret.trivias).build(program);
+
+        let adapter = Adapter {
+            path_components: vec![Some("index".to_string())],
+            semantic: Rc::new(semantic_ret.semantic),
+        };
+
+        let args: BTreeMap<Arc<str>, FieldValue> = BTreeMap::new();
+
+        let adapter = Arc::from(&adapter);
+
+        #[allow(clippy::items_after_statements)]
```

Comment:
I think these are allow by default but CI doesn't see it that way for some reason...

Reply:
This is lint is in the pedantic group: https://rust-lang.github.io/rust-clippy/master/index.html#/clippy::items_after_statements

Pedantic lints seem to be enabled in the `.cargo/config.toml` so this seems to be working as expected, unless I'm missing something?